contracts/seed/Seed.sol

```diff
@@ -321,14 +319,13 @@ contract Seed {
 
         if (!isFunded) {
             require(
-                // classSeedAmountRequired is an amount which is needed to be sold
-                // So when it's reached, for others will their balance be bigger or not - doesn't matter anymore.
                 seedToken.balanceOf(address(this)) >= seedAmountRequired,
                 "Seed: sufficient seeds not provided"
             );
             isFunded = true;
         }
 
+        // fundingAmount is an amount of fundingTokens required to buy _seedAmount of SeedTokens
         uint256 seedAmount = (_fundingAmount * PRECISION) / price;
```

Comment:
this comment is confusing:
- misses `_` in `fundingAmount` (because right side has `_`)
- same for `seedAmount`

What is this comment trying to clarify?

Reply:
So this is a comment that I copied back in from the version Doaism had sent to the audits.

I went over the contract to check how many lines I have changed that I could change back because the changed lines count is 50% higher than I've estimated.

I ended up deciding against adding lines again but this one was left in. I remove it